api/instance.go

```diff
@@ -103,9 +103,6 @@ type InstancePatch struct {
 	Name          *string `jsonapi:"attr,name"`
```

Comment:
Above in the instance, we can also remove the following fields, at least not returning back to the frontend. We can do it in a follow-up PR though.

```
// Instance is the API message for an instance.
type Instance struct {
	Host          string  `jsonapi:"attr,host"`
	Port          string  `jsonapi:"attr,port"`
	// Database is the initial connection database for PostgreSQL only.
	Database string `jsonapi:"attr,database"`
	Username string `jsonapi:"attr,username"`
	// Password is not returned to the client
	Password string
}
```